web/views/shared_view.ex

```diff
@@ -275,12 +283,33 @@ defmodule CoursePlanner.SharedView do
   end
 
   def class_list(classes, opts \\ []) do
-    empty_text = opts[:empty_text] || "There are no users here yet"
+    empty_text = opts[:empty_text] || "There are no classes here yet"
     show_attendances = opts[:show_attendances] || false
 
     render "class_list.html", classes: classes,
                               empty_text: empty_text,
                               show_attendances: show_attendances
   end
 
+  def course_list(offered_courses, opts \\ []) do
+    empty_text = opts[:empty_text] || "There are no courses here yet"
+
+    render "course_list.html", offered_courses: offered_courses,
+                               empty_text: empty_text
+  end
+
+  def task_list(tasks, opts \\ []) do
+    empty_text = opts[:empty_text] || "There are no tasks here yet"
+
+    render "task_list.html", tasks: tasks,
+                             empty_text: empty_text
+  end
+
+  def holiday_list(holidays, opts \\ []) do
+    empty_text = opts[:empty_text] || "There are no holidays here yet"
+
+    render "holiday_list.html", holidays: holidays,
+                                empty_text: empty_text
```

Comment:
Yes. It has a default empty text. It can be specialized (e. g. this student has no course yet), but I can't tell if it is necessary. It's rather a UX question.